examples/llama/train_llama2.sh

```diff
@@ -0,0 +1,323 @@
+#!/bin/bash
+###############################################################################
+#################################################################################
+# set -x
+
+# set envs 
+export GPU_MAX_HW_QUEUES=2
+export TORCH_NCCL_HIGH_PRIORITY=1
+export NCCL_CHECKS_DISABLE=1
+export NCCL_IB_HCA=rdma0,rdma1,rdma2,rdma3,rdma4,rdma5,rdma6,rdma7 
+export NCCL_IB_GID_INDEX=3
+export NCCL_CROSS_NIC=0
+export NCCL_SOCKET_IFNAME=ens50f0np0 # network interface  
+export GLOO_SOCKET_IFNAME=ens50f0np0 # network interface
+export CUDA_DEVICE_MAX_CONNECTIONS=1
+export NCCL_PROTO=Simple
+export RCCL_MSCCL_ENABLE=0
+export TOKENIZERS_PARALLELISM=false
+export HSA_NO_SCRATCH_RECLAIM=1
+
+
+# parsing input arguments
+for ARGUMENT in "$@"
+do
+   KEY=$(echo $ARGUMENT | cut -f1 -d=)
+
+   KEY_LENGTH=${#KEY}
+   VALUE="${ARGUMENT:$KEY_LENGTH+1}"
+
+   export "$KEY"="$VALUE"
+done
+
+
+TIME_STAMP=$(date +"%Y-%m-%d_%H-%M-%S")
+EXP_NAME="${EXP_NAME:-perf}"
+
+TEE_OUTPUT="${TEE_OUTPUT:-1}"
+#NO_TORCH_COMPILE="${NO_TORCH_COMPILE:-1}"
```

Comment:
please remove the commented line  NO_TORCH_COMPILE.  we don't need this option anyways.